spring-integration-webflux/src/main/java/org/springframework/integration/webflux/config/WebFluxOutboundChannelAdapterParser.java

```diff
@@ -79,6 +79,17 @@ else if (hasTypeExpression) {
 							.getBeanDefinition());
 		}
 
+		String attributeVariablesExpression = element.getAttribute("attribute-variables-expression");
+		boolean hasAttibuteVariablesExpression = StringUtils.hasText(attributeVariablesExpression);
+
+		if (hasAttibuteVariablesExpression) {
+			builder.addPropertyValue("attributeVariablesExpression",
+					BeanDefinitionBuilder.rootBeanDefinition(ExpressionFactoryBean.class)
```

Comment:
See `IntegrationNamespaceUtils.createExpressionDefIfAttributeDefined()` instead for convenience.

Reply:
Code ha been updated to use IntegrationNamespaceUtils.createExpressionDefIfAttributeDefined()

Reply:
@saxenaj ,

you know, there is no reason to add this kind of comments if you agree with us: we just simply will see that in the next commit you add to the PR.
Otherwise I got at a bout 10 emails from you instead of just one about a new commit. 😄